cluster-operator/src/test/java/io/strimzi/operator/cluster/ClusterOperatorConfigTest.java

```diff
@@ -79,7 +79,7 @@ public void testEnvVarsDefault() {
     public void testListOfNamespaces() {
 
         Map<String, String> envVars = new HashMap<>(ClusterOperatorConfigTest.envVars);
-        envVars.put(ClusterOperatorConfig.STRIMZI_NAMESPACE, "foo, bar ,, baz , ");
+        envVars.put(ClusterOperatorConfig.STRIMZI_NAMESPACE, "foo,bar,baz");
```

Comment:
Can we test a case with whitespaces too: `" foo, bar , baz "

Reply:
No, because we do not support it anymore. It doesn't pass the regex. I'm not sure I understand why would we have special handling for different typos in the configuration. But if you think we should, I would need to adapt the validation for it.